components/datadog/kubernetesagentparams/params.go

```diff
@@ -67,6 +67,21 @@ func NewParams(options ...Option) (*Params, error) {
 	return common.ApplyOption(version, options)
 }
 
+// WithClusterName sets the name of the cluster. Should only be used if you know what you are doing. Must no be necessary in most cases.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Could help adding more context, for example with a use case where this is used